server/api/db/base.py

```diff
@@ -1068,3 +1068,106 @@ def get_time_window_tracker_record(
         self, session, key: str, raise_on_not_found: bool = True
     ):
         pass
+
+    def store_model_endpoint(
+        self,
+        session,
+        model_endpoint: mlrun.common.schemas.ModelEndpointV2,
+        name: str,
+        project: str,
+    ) -> str:
+        pass
+
+    def get_model_endpoint(
+        self,
+        session,
+        project: str,
+        name: str,
+        uid: str = None,
+    ) -> mlrun.common.schemas.ModelEndpointV2:
+        """
+        Get a model endpoint by project, name and uid.
+        If uid is not provided, the latest model endpoint with the provided name and project will be returned.
+
+        :param session: The database session.
+        :param project: The project name.
+        :param name:    The model endpoint name.
+        :param uid:     The model endpoint uid.
+        :return:        The model endpoint object.
+        """
+        pass
+
+    def update_model_endpoint(
+        self,
+        session,
+        project: str,
+        name: str,
+        attributes: dict,
+        uid: str = None,
```

Comment:
just to keep consistency with other methods (also relevant to `get_model_endpoint`)
```suggestion
        uid: typing.Optional[str] = None,
```